applications/RomApplication/tests/structural_dynamic_test_files/test_ROM.py

```diff
@@ -27,10 +27,11 @@ def test_Struct_Dynamic_ROM_2D(self):
                 UP=0
                 DOWN=0
                 for j in range((np.shape(ObtainedOutput)[0])):
-                    UP += (NodalArea[j]*(    (ExpectedOutput[j,i] - ObtainedOutput[j,i]   )**2)  )
-                    DOWN +=  NodalArea[j]
-                L2 = np.sqrt(UP/DOWN)
-                self.assertLess(L2, 1.0e-4)
+                    if ExpectedOutput[j,i] != 0:
+                        UP += (NodalArea[j]*(    (1  - (ObtainedOutput[j,i] / ExpectedOutput[j,i] )   )**2)  )
+                        DOWN +=  NodalArea[j]
+                L2 = (np.sqrt(UP/DOWN)) *100
+            self.assertLess(L2, 0.5) #percent
```

Comment:
? you now set the tolerance very large! it should be very very small!!

Reply:
Tolerances have been tighten up. These are percentual, so we are now acceptig an error of 0.35% in the dynamic ROM simulation...

Reply:
but it is a tolerance wrt the FOM, or wrt the reference solution? the point is that if it is versus the reference it should be essentially zero